templates/html/search.js

```diff
@@ -156,7 +156,7 @@ function SearchBox(name, resultsPath, extension)
 
   this.OnSearchSelectHide = function()
   {
-    this.hideTimeout = setTimeout(this.name +".CloseSelectionWindow()",
+    this.hideTimeout = setTimeout(this.Search.bind(this),
                                   this.closeSelectionTimeout);
```

Comment:
I have my doubts that this is correct as when on selects the magnifying glass in the search box we get a drop-down box with e.g "All" / "Classes" / "Namespaces" / "Functions" / ... and this drop-down box remains when it looses focus whilst in the current version the drop-down box vanishes.

Reply:
yeah you're right, let me investigate a bit

Reply:
alright updated the PR and tested for that test case, that drop down box should now disappear when not in focus.